gcc/rust/privacy/rust-reachability.cc

```diff
@@ -31,6 +31,47 @@ maybe_get_vis_item (std::unique_ptr<HIR::Item> &item)
   return static_cast<HIR::VisItem *> (item.get ());
 }
 
+ReachLevel
+ReachabilityVisitor::initial_reach (const HIR::Visibility &item_visibility)
+{
+  if (item_visibility.is_public ())
```

Comment:
might be a good case to use a ternary statement here, if there is more logic to add then leave it as is.